tests/std/tests/P1222R4_flat_set/test.cpp

```diff
@@ -65,10 +65,52 @@ void assert_reversible_container_requirements(const T& s) {
     static_assert(is_convertible_v<typename T::reverse_iterator, typename T::const_reverse_iterator>);
 }
 
+template <class T>
+void test_ebco() {
+    using container_type = T::container_type;
+    if constexpr (is_empty_v<typename T::key_compare>) {

```

Comment:
![image](https://github.com/microsoft/STL/assets/60953653/a934a6bf-1bc0-447f-ada6-b5eb7a149e44)
`_Compressed_pair`'s ebco is limited by final-ness; however I think it ok not to add `is_final_v` here, as long as we don't bother testing `final` comparators.